### DISCUSSION TITLE
جلسه پنجم دوره بهترین مسیر ورود به دنیای برنامه‌نویسی

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#179] [Webinar & Meetups]
# جلسه پنجم دوره بهترین مسیر ورود به دنیای برنامه‌نویسی

`در تاریخ: ۱۲-۵-۱۴۰۱`

### سلام و عرض ادب، خیلی خوش‌آمدید

لطفا اگر مطلب مفیدی در رستای محتوای مطرح شده دارید همراه با تمرین‌های این جلسه در اینجا مطرح کنید.

[دریافت ویدیو وبینار](https://eseminar.tv/wb77037)

**مباحت مهم مطرح شده:**

۱. بررسی وضعیت شما

۲. مشارکت در پروژه‌های گیت‌هاب

۳. ادامه مباحث HTML

۴. ادامه ساخت اولین وبسایت شخصی در گیت‌هاب

۵. ادامه مفاهیم پایه و بررسی لینک‌های جانبی و مطالب تکمیلی جلسه قبل

**[مطالب بیشتر در کانال تلگرام](https://t.me/AliMDSchool)**

**Comment by alimd** [comment]
## روند آموزشی و تمرینات که باید طی کنید

| کد       |                                                                                   توضیح |
| -------- | --------------------------------------------------------------------------------------: |
| **W0**   |                        [وبینار ورود به دنیای برنامه‌نویسی](https://eseminar.tv/wb66591) |
| -        |                                                                                       - |
| **W1**   | [وبینار بهترین مسیر ورود به دنیای برنامه‌نویسی (جلسه اول)](https://eseminar.tv/wb73140) |
| **P1**   |                     [تمرین وبینار اول: ساخت اولین بازی](https://www.aparat.com/v/5iH73) |
| **V1**   |           [ویدیو تکمیلی: مقدمه‌ای بر آموزش جاوااسکریپت](https://www.aparat.com/v/grw2Y) |
| -        |                                                                                       - |
| **V2**   |             [ویدیو تکمیلی: دوره‌همی دنیای برنامه‌نویسی](https://www.aparat.com/v/a9h0M) |
| **W2**   |                                                               ...وبینار دوم بهترین مسیر |
| **P2.1** |                                   تمرین وبینار دوم 1: ساخت ریپو گیت‌هاب و زبان مارک‌دان |
| **P2.2** |                           تمرین وبینار دوم 2: تست بیشتر زبان مارک‌دان همراه لینک و جدول |
| -        |                                                                                       - |
| **W3**   |                                                               ...وبینار سوم بهترین مسیر |
| **P3.1** |                                        W3Schools / HTML تمرین وبینار سوم 1: مرور مقدمات |
| **P3.2** |                              JSBin تمرین وبینار سوم 2: نوشتن اولین کدها و ارسال لینک از |
| -        |                                                                                       - |
| **W4**   |                              [...وبینار چهارم بهترین مسیر](https://eseminar.tv/wb76239) |
| **P4**   |                                  تمرین وبینار چهارم: ساخت یک سایت شخصی اولیه در گیت‌هاب |
| -        |                                                                                       - |
| **W5**   |                               [...وبینار پنجم بهترین مسیر](https://eseminar.tv/wb77037) |

## ارزیانی وضعیت شما در دوره

| User                 |              W0               |              W1               |              P1               |              V1               |              V2               |              W2               |             P2.1              |             P2.2              |              W3               |             P3.1              |             P3.2              |              W4               |              P4               |              W5               |
| -------------------- | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: | :---------------------------: |
| @adltalab            |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @MKMoghadasan        |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @sabermanesh         |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @shosseinfarjadi     |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @amiresp             |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @MuhammadMGH         |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @mahshidhooshmand    |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |
| @alirezaFAkhar       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @sepehrislami        |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: | :negative_squared_cross_mark: |               ?               |
| @amir91179           |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @kianooshso          | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: |               ?               |
| @abolfazlrasoulizade |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @satafti             |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |              :x:              |      :white_check_mark:       |              :x:              |              :x:              |      :white_check_mark:       |              :x:              |               ?               |
| @mahyasbm            |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @marihooshmand       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |              :x:              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @Rahimpanah          |      :white_check_mark:       |      :white_check_mark:       |              :x:              |              :x:              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |               ?               |
| @A-sina              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |
| @amirmmolazemi       |              :x:              |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |
| @berenjian           | :negative_squared_cross_mark: | :negative_squared_cross_mark: |              :x:              |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: |              :x:              |              :x:              | :negative_squared_cross_mark: |      :white_check_mark:       | :negative_squared_cross_mark: |
| @MohamadAliAlizadeh  |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: | :negative_squared_cross_mark: |
| @KianooshSoleimani   |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |              :x:              |              :x:              |              :x:              |              :x:              |              :x:              |              :x:              |               -               |
| @M-beheshti          |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |      :white_check_mark:       |              :x:              |      :white_check_mark:       |           :warning:           |               -               |

لطفا برای تکمیل این جدول کد مارک‌دان زیر را کپی و برای خودتون تکمیل و در کامنت همین پیام ارسال کنید.

```md
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @?   | :x: | :x: | :x: | :x: | :x: | :x: | :x:  | :x:  | :x: | :x:  | :x:  | :x: | :x: |  -  |
```

علامت :negative_squared_cross_mark: با کد `:negative_squared_cross_mark:`  
برای وبینارها به معنی حضور در وبینار بدون دریافت و مشاهده دقیق ویدیو (تاخیر زیاد قابل قبول نیست و با غیبت فرقی نداره مگر بعد از مشاهده دقیق ویدیو)

علامت :white_check_mark: با کد `:white_check_mark:`  
برای وبینارها به معنی دانلود و مشاهده دقیق ویدیو وبینار (حضور تنها قابل قبول نیست و با حضور ساده فرقی ندارد)
برای تمرین‌ها به انجام کامل و ارسال لینک در کامنت پست همان جلسه در گیت‌هاب (خطا در کل یا انجام ناقص قابل قبول نیست و با انجام ندادن فرقی ندارد)

علامت :warning: با کد `:warning:`  
برای تمرین‌ها و به معنای خطا یا مشکل در انجام کار (اگر هیچ سوالی پرسیده نشده قابل قبول نیست و با انجام ندادن فرقی ندارد)
لطفا حتما مشکل را در گروه کلاس و یا در کامنت پست جلسه در گیت‌هاب مطرح کنید تا من یا دیگر دوستان کلاس بتونیم بهتون کنیم.

**Reply by amirmmolazemi** [reply]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @amirmmolazemi  | ✅  | ✅  | ✅  | ✅  | ✅  | ✅  | ✅   | ✅   | ✅  | ✅   | ✅   | ✅  | ✅  |  ✅   |

**Reply by KianooshSoleimani** [reply]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @KianooshSoleimani   | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark: | :white_check_mark: |  :white_check_mark:  |

updated h☝️

**Reply by Rahimpanah** [reply]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @Rahimpanah   | ✅  | ✅  | ❌ | ❌ | ✅  | ✅ | ✅  |✅  | ✅ | ✅  | ✅  | ✅ | ✅ |  ✅ |

**Reply by shosseinfarjadi** [reply]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @shosseinfarjadi    |✅ | ✅ | ✅ |✅ | ✅| ✅| ✅ | ✅  |✅ | ✅  | ✅  | ✅ | ✅ |  ✅
  |

**Reply by FatemeGharavian** [reply]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| ---- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |
| @FatemeGharavian  |:white_check_mark:| :x: | :x: | :x: | :white_check_mark: | :white_check_mark: | :white_check_mark:  | :x:  | :white_check_mark: | :-:  | :white_check_mark: | :x: | :white_check_mark: |  -  |

**Comment by MKMoghadasan** [comment]
```
<html>
<head>
</head>
<body style="background-color:powderyellow;">
  <h1 title="چرا ناراحت میشی" style="font-size:120px;" style="color:powderblue" style="text-align:center;"> 😉سلام <br/> چطورید؟ </h1>
  <hr>
  
  <a style="text-align:center;" href="https://shaaer.net//">sait</a>
  <hr>
  
  <img src="https://shaaer.net/wp-content/uploads/2022/03/558A4097-copy-scaled.jpg" width=700 height=800>
</body>  
<html>
```

**Comment by shosseinfarjadi** [comment]
<html>
<head>
</head>
<h2>سلام</h2>
<p>خوبین</p>
<!DOCTYPE html>
<html>
<head>
</head>
<body>

<h1>hi</h1>
<p>how are you</p>

</body>
</html>

**Reply by shosseinfarjadi** [reply]
تکلیف

**Comment by shosseinfarjadi** [comment]
https://shosseinfarjadi.github.io/1.html
سایت با عکس

**Reply by alimd** [reply]
خوبه حسن جان
حالا ببین میتونی خوشگلش کنی

**Comment by amiresp** [comment]
این جلسه و جلسه بعد (فکر کنم)
https://amiresp.github.io/

**Reply by alimd** [reply]
خیلی عالی شده 👍🏻

**Comment by amir91179** [comment]
سلام استاد 
https://amir91179.github.io/

**Reply by alimd** [reply]
خوبه امیر جان
حالا ببین میتونی خوشگلش کنی

**Reply by amir91179** [reply]
سلام تمرین دومش
https://amir91179.github.io/Profile-page.html

**Comment by MohamadAliAlizadeh** [comment]
Salam

**Reply by alimd** [reply]
مرسی که همت کردی و ارسال کردی 
خوشحالم کردی

**Comment by A-sina** [comment]
# tamrin jalase 5om(site baad az jalase 5om)
[site](//a-sina.github.io)

**Reply by alimd** [reply]
احسنت
فقط اون زیر خط آبی آیکن هارو حذف کن
```
a {
  text-decoration: none;
}
```

**Reply by A-sina** [reply]
anjam shod 
mamnoon

**Comment by amirmmolazemi** [comment]


**Reply by alimd** [reply]
احسنت
ببین متونی عکس رو بیاری خط جدا و وسط باشه و گردش کنی؟

**Reply by amirmmolazemi** [reply]
### salam dobare ostad
### befarmayid
#### [site](https://amirmmolazemi.github.io/)

**Comment by alirezaFAkhar** [comment]
سلام استاد خسته نباشید  تمرین جلسه پنج 
https://alirezafakhar.github.io

**Reply by MohammadHoseynSeyedi85** [reply]
سلام این انیمیشن هاتون رو چطور گذاشتید؟

**Comment by sabermanesh** [comment]
[site](https://sabermanesh.github.io)
چرا نمیاد وسط ؟

**Comment by AbolfazlRasoulizade** [comment]
باید به خود body دستور بدی

**Comment by MKMoghadasan** [comment]
MKMoghadasan.github.io

**Reply by M-Beheshti** [reply]
سلام 
https://jsbin.com/widaqid/edit?html,css,output

**Comment by Rahimpanah** [comment]
Hello teacher,
This is my exercise for session 5:
https://output.jsbin.com/vilexuj/

**Comment by amirmmolazemi** [comment]
### salam o dorood ostad
#### tamrin jalase 6 om 
[ site](https://amirmmolazemi.github.io/)

**Comment by sepehrislami** [comment]
سلام و عرض ادب
[my practice](https://sepehrislami.github.io)

**Comment by sepehrislami** [comment]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |P5 |W6|P6|
| --- | :-: | :-: | :-: | :-: | :-: | :-: | :--: | :--: | :-: | :--: | :--: | :-: | :-: | :-: |:-: |:-:|:--:
| @sepehrislami   | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark: | :white_check_mark: |  :white_check_mark:|:white_check_mark:|❌|❌|

**Comment by mahshidhooshmand** [comment]
سلام
[تمرین جلسه پنجم](https://output.jsbin.com/jagoqow)

**Comment by shosseinfarjadi** [comment]
https://output.jsbin.com/sugiqey/1
فعلا اینو زدم یکی زدم خوب شده بود متاسفانه پاک شد دارم درستش میکنم

**Comment by FatemeGharavian** [comment]
https://jsbin.com/naxisiquqa/1/edit?html,css,js,output

**Comment by amirmmolazemi** [comment]
# salam ostad  mihandoost
## taklif takmil site ba komak video css.stricks
### [project](https://github.com/Amirmmolazemi/web-intoduction)

**Comment by shosseinfarjadi** [comment]
https://output.jsbin.com/goyeriv/8
سلام تکلیف

**Comment by A-sina** [comment]
salam o dorood 
[site](https://a-sina.github.io) pas az video [jenab seyedi](https://css-tricks.ir/%D8%AF%D9%88%D8%B1%D9%87-%D9%85%D9%82%D8%AF%D9%85%D8%A7%D8%AA%DB%8C-%D8%B7%D8%B1%D8%A7%D8%AD%DB%8C-%D9%88%D8%A8/)

**Comment by amirmmolazemi** [comment]
salam o dorood ostad Mihandoost
takmil [site](https://amirmmolazemi.github.io/index.html) pas az didan [video](https://css-tricks.ir/%D8%AF%D9%88%D8%B1%D9%87-%D9%85%D9%82%D8%AF%D9%85%D8%A7%D8%AA%DB%8C-%D8%B7%D8%B1%D8%A7%D8%AD%DB%8C-%D9%88%D8%A8/)  jenab seyedi

**Comment by shosseinfarjadi** [comment]
https://output.jsbin.com/jovupej/2
سلام سایت جدید فقط برای کلید گذاشتن discord و telegramمشکل خوردم
نمیدونم چرا⁈⁇‽⁇⁈⁇⁈‽¿¿

**Reply by A-sina** [reply]
dar tag haye img be src deghat konid eshtebah vared shode

**Reply by adltalab** [reply]
همونطور که @A-sina فرمودند، آدرس تصاویر مشکل داره ...
یک اصلاحیه کوچیک رو کد شما انجام دادم، شاید به درد بخوره..
https://jsbin.com/fibacofumi

**Reply by shosseinfarjadi** [reply]
سلام جناب عدل طلب ممنون از شما مونده بودم چیه مشکلش خیلی ممنون😘🥰❤❤❤

**Comment by shosseinfarjadi** [comment]
https://output.jsbin.com/zokariw/2
سلام پروژه اصلاح شده
ممنون از همه بزرگوارانی که کمک کردن❤❤

**Reply by adltalab** [reply]
بهتره برای بخش `projects` از padding استفاده کنید،   *متن و تصاویر از راست و چپ به گوشه ها چسبیده*

For example:
```
.projects {
  padding: 2% 5%;
}
```

**Reply by shosseinfarjadi** [reply]
ممنون چشم

**Comment by MohammadHoseynSeyedi85** [comment]
| User | W0  | W1  | P1  | V1  | V2  | W2  | P2.1 | P2.2 | W3  | P3.1 | P3.2 | W4  | P4  | W5  |
| :white_check_mark: |:white_check_mark:|:white_check_mark: | :white_check_mark: |:white_check_mark:|:white_check_mark: |:white_check_mark: | :white_check_mark: | :white_check_mark: |:white_check_mark: |:white_check_mark:| :white_check_mark: |:white_check_mark:  | :white_check_mark:|:white_check_mark: |

**Comment by mydavid66** [comment]
با سلام و وقت بخیر
تمرین وب سایت شخصی 
https://mydavid66.github.io/

**Comment by armanasadian** [comment]
سلام 

https://jsbin.com/hiwudeh/5/edit?html,css,output

**Comment by AmirrezaTN** [comment]
Salam arz adab
[website](https://amirrezatn.github.io/)
code
```html <!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width">
  <title>Amirreza Taghavinia</title>
</head>
<body>
  <h1>Amirreza TN</h1>
  <img     src="https://github.com/AmirrezaTN/AmirrezaTN.github.io/blob/main/assets/images/photo_2023-11-14_11-09-51.jpg?raw=true" alt="my picture" width="120" height="120"/>
  <p>Hi my name is Amirreza Taghavinia. I'm learning web developing and I'm kind of c++ junior.</p>
  <img src="https://github.com/AmirrezaTN/AmirrezaTN.github.io/blob/main/assets/images/Telegram-Logo.png?raw=true" alt="telegram icon" width="60" height="32"/>
  <a href="https://t.me/AmirrezaTaghavinia">Telegram link </a>
  <br/>
  <svg height="32" aria-hidden="true" viewBox="0 0 16 16" version="1.1" width="32" data-view-component="true" class="octicon octicon-mark-github v-align-middle color-fg-default">
    <path d="M8 0c4.42 0 8 3.58 8 8a8.013 8.013 0 0 1-5.45 7.59c-.4.08-.55-.17-.55-.38 0-.27.01-1.13.01-2.2 0-.75-.25-1.23-.54-1.48 1.78-.2 3.65-.88 3.65-3.95 0-.88-.31-1.59-.82-2.15.08-.2.36-1.02-.08-2.12 0 0-.67-.22-2.2.82-.64-.18-1.32-.27-2-.27-.68 0-1.36.09-2 .27-1.53-1.03-2.2-.82-2.2-.82-.44 1.1-.16 1.92-.08 2.12-.51.56-.82 1.28-.82 2.15 0 3.06 1.86 3.75 3.64 3.95-.23.2-.44.55-.51 1.07-.46.21-1.61.55-2.33-.66-.15-.24-.6-.83-1.23-.82-.67.01-.27.38.01.53.34.19.73.9.82 1.13.16.45.68 1.31 2.69.94 0 .67.01 1.3.01 1.49 0 .21-.15.45-.55.38A7.995 7.995 0 0 1 0 8c0-4.42 3.58-8 8-8Z"></path>
</svg>
  <a href="https://github.com/AmirrezaTN">Github link </a>
  <br/>
  <img src="https://github.com/AmirrezaTN/AmirrezaTN.github.io/blob/main/assets/images/64779-computer-design-email-icon-icons-free-transparent-image-hd.png?raw=true" alt="email image" width="35" height="35" />
  <a>amooreza.t1382@gmail.com</a>
</body>
</html>  ```
